test/common/http/http2/codec_impl_test.cc

```diff
@@ -62,6 +62,72 @@ class Http2CodecImplTest : public testing::TestWithParam<uint64_t> {
   ConnectionWrapper server_wrapper_;
 };
 
+TEST_P(Http2CodecImplTest, ExpectContinueHeadersOnlyResponse) {
+  MockStreamDecoder response_decoder;
+  StreamEncoder& request_encoder = client_.newStream(response_decoder);
+
+  MockStreamDecoder request_decoder;
+  StreamEncoder* response_encoder;
+  EXPECT_CALL(server_callbacks_, newStream(_))
```

Comment:
Suggest refactoring this encoder/decoder setup into the Http2CodecImplTest class, it seems the boiler plate is now in quite a few test cases.